docs/02_stratum0_client.md

```diff
@@ -1,136 +1,290 @@
+### 2.1.7 Cron job for resigning the whitelist
 
-If you just keep the master key on our Stratum 0 sever, you can set up a simple cronjob for resigning the whitelist. For instance, make a file `/etc/cron.d/cvmfs_resign` with the following content to do this every Monday at 11:00:
+Each CernVM-FS repository has a whitelist containing fingerprints of certificates that are allowed to sign the repository.
+This whitelist has an expiration time of (by default) 30 days, so you regularly have to resign the whitelist.
+
+There are several ways to do this, see for instance [the page about master keys](https://cvmfs.readthedocs.io/en/stable/cpt-repo.html#sct-master-keys)
+in the CernVM-FS documentation.
+
+If you keep the master key on our Stratum 0 sever, you can set up a simple cronjob for resigning the whitelist. For instance, make a file `/etc/cron.d/cvmfs_resign` with the following content to do this every Monday at 11:00:
 ```
 0 11 * * 1 root /usr/bin/cvmfs_server resign repo.organization.tld
 ```
 
-### Remove a repository
-An existing repository can be removed by running:
+For the sake of this tutorial this is not really necessary of course, but it's an important aspect to be aware of.
+
+### 2.1.8 Removing a repository
+
+An existing repository can be removed easily by running:
 ```
 sudo cvmfs_server rmfs repo.organization.tld
 ```
 
+Obviously you should only do this when you actually *want* to get rid of the repository...
 
-## Set up a client
-Accessing CernVM-FS repositories on a client machine involves three steps: installing the CernVM-FS client package, adding some configuration files for the repository you want to connect to, and finally run a CernVM-FS setup procedure that will mount the repository.
 
-Since the client is going to pull in files over an HTTP connection, you need sufficient space for storing a local cache on the client machine. You can define the maximum size of your cache in the settings; the larger your cache is, the less often you have to pull in files again, and the faster your applications will start. Note that you can add more cache layers by adding a proxy nearby your client; this will be covered in a later section.
+## 2.2 Setting up a client
 
-### Installation
-The installation is the same as for the Stratum 0, except that you only need the `cvmfs` package:
+Accessing a CernVM-FS repository on a client system involves three steps:
+
+1. installing the CernVM-FS client package;
+- adding some configuration files for the repository you want to connect to;
+- running the CernVM-FS setup procedure that will mount the repository.
```

Comment:
```suggestion
1. installing the CernVM-FS client package;
2. adding some configuration files for the repository you want to connect to;
3. running the CernVM-FS setup procedure that will mount the repository.
```

Reply:
This is a trick to not having the number them ourselves (it looks fine in the rendered version with 1, 2, 3), but it looks confusing in the source file, so makes sense to change.